src/js/components/ToggleGroup/stories/simple.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Box, ToggleGroup } from 'grommet';
+
+export const Simple = () => (
+  <Box gap="large" pad="large">
+    <ToggleGroup options={['Option 1', 'Option 2', 'Option 3']} />
```

Comment:
Replace this story with the "choose view" execution?

Reply:
Done.